pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -425,7 +424,7 @@ func (plugin *cniNetworkPlugin) buildCNIRuntimeConf(podName string, podNs string
 		portMappingsCapability: portMappingsParam,
 	}
 
-	ingress, egress, err := bandwidth.ExtractPodBandwidthResources(annotations)
+	ingress, ingressBurst, egress, egressBurst, err := bandwidth.ExtractPodBandwidthResources(annotations)
```

Comment:
Do we also need to update the usage of this method in `pkg/kubelet/dockershim/network/kubenet/kubenet_linux.go`?

Reply:
yes